octobot_evaluators/evaluators/TA_evaluator.py

```diff
@@ -81,18 +81,22 @@ async def evaluators_callback(self,
                                   time_frame,
                                   data):
         # Used to communicate between evaluators
+        # ignore time frames related to other instances if case of a non time frame wildcard evaluator
+        time_frames_to_update = data.get(constants.EVALUATOR_CHANNEL_DATA_TIME_FRAMES, []) \
+            if self.get_is_time_frame_wildcard() else \
+            [tf for tf in data.get(constants.EVALUATOR_CHANNEL_DATA_TIME_FRAMES, []) if tf is self.time_frame]
```

Comment:
`if tf is self.time_frame` so you just want to have a list of self.time_frame ?

Reply:
is, but only if self.time_frame is in constants.EVALUATOR_CHANNEL_DATA_TIME_FRAMES

Reply:
I mean that I don't understand why you want a list of same values like : [self.time_frame, self.time_frame, self.time_frame...]